stubs/frozendict.pyi

```diff
@@ -14,6 +14,8 @@
 
 # Stub for frozendict.
 
+from __future__ import annotations
```

Comment:
We do seem to actually need these, I'm not sure why `flake8` was running OK without them. Maybe it wasn't even checking `*.pyi` files?